database/mssql/scripts/versions/v_41_ddl.sql

```diff
@@ -0,0 +1,250 @@
+SET ANSI_NULLS ON
+GO
+
+SET QUOTED_IDENTIFIER ON
+GO
+
+SET NOCOUNT ON
+GO
+
+SET XACT_ABORT ON
+GO
+
+SET TRANSACTION ISOLATION LEVEL SERIALIZABLE
+GO
+
+BEGIN TRANSACTION
+GO
+
+IF @@ERROR <> 0
+   SET NOEXEC ON
+GO
+    DROP TABLE [permit].[ORBC_LOA_PERMIT_TYPE_DETAILS]
+    DROP TABLE [permit].[ORBC_LOA_VEHICLES]
+    DROP TABLE [permit].[ORBC_LOA_DETAILS]
+    DROP SEQUENCE [permit].[ORBC_LOA_NUMBER_SEQ]
+    
+    CREATE SEQUENCE permit.ORBC_LOA_NUMBER_SEQ
+    START WITH 100000
+    INCREMENT BY 1 ;
+
+CREATE TABLE [permit].[ORBC_LOA_DETAILS] (
+   [LOA_ID] [int] IDENTITY(1,1) NOT NULL,
+   [LOA_NUMBER] [int] NOT NULL  DEFAULT (NEXT VALUE FOR permit.ORBC_LOA_NUMBER_SEQ),
+   [COMPANY_ID] [int] NOT NULL,
+   [REVISION] [tinyint] NULL,
+   [PREVIOUS_LOA_ID] [tinyint] NULL,
+   [ORIGINAL_LOA_ID] [tinyint] NOT NULL,
```

Comment:
Should be int as it references LOA_ID.

ALTER TABLE [permit].[ORBC_LOA_DETAILS]  
   WITH CHECK ADD CONSTRAINT [FK_ORBC_LOA_DETAILS_ORIGINAL_LOA_ID] 
   FOREIGN KEY([ORIGINAL_LOA_ID])
   REFERENCES [permit].[ORBC_LOA_DETAILS] ([LOA_ID])
GO
ALTER TABLE [permit].[ORBC_LOA_DETAILS] CHECK CONSTRAINT [FK_ORBC_LOA_DETAILS_ORIGINAL_LOA_ID]